src/api/spec/models/obs_factory/distribution_spec.rb

```diff
@@ -1,17 +1,229 @@
 require 'rails_helper'
+require 'webmock/rspec'
 
 RSpec.describe ObsFactory::Distribution do
+  let(:project) { create(:project, name: 'openSUSE:Factory') }
+  let(:other_project) { create(:project, name: 'other_project') }
+  let!(:distribution) { ObsFactory::Distribution.new(project) }
+
   describe '::new' do
     def strategy_for(name)
       ObsFactory::Distribution.new(create(:project, name: name)).strategy
     end
 
-    it { expect(strategy_for('openSUSE:Factory')).to         be_kind_of ObsFactory::DistributionStrategyFactory }
+    it { expect(distribution.strategy).to                    be_kind_of ObsFactory::DistributionStrategyFactory }
     it { expect(strategy_for('openSUSE:Factory:PowerPC')).to be_kind_of ObsFactory::DistributionStrategyFactoryPPC }
     it { expect(strategy_for('openSUSE:42.3')).to            be_kind_of ObsFactory::DistributionStrategyOpenSUSE }
     it { expect(strategy_for('SUSE:SLE-12-SP1:GA')).to       be_kind_of ObsFactory::DistributionStrategySLE12SP1 }
     it { expect(strategy_for('SUSE:SLE-15:GA')).to           be_kind_of ObsFactory::DistributionStrategySLE15 }
     it { expect(strategy_for('SUSE:SLE-15-SP1:GA')).to       be_kind_of ObsFactory::DistributionStrategySLE15 }
     it { expect(strategy_for('SUSE:SLE-12-SP3:Update:Products:CASP20')).to be_kind_of ObsFactory::DistributionStrategyCasp }
   end
+
+  describe 'self.attributes' do
+    it { expect(ObsFactory::Distribution.attributes.class).to eq(Array) }
+    it { expect(ObsFactory::Distribution.attributes.count).to eq(13) }
```

Comment:
What is this testing? I find the 13 a bit magic here

Reply:
I agree with @coolo, why 13? I also don't understand what this test is really testing... :confused:

Reply:
Same here

Reply:
Actually, I don't think we need a test for this. Since [attributes](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/obs_factory/distribution.rb#L24) is just a static array.